### PR TITLE
Create 2023-05-01-Level-Finance.md

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

File: content/attacks/posts/2023-05-01-Level-Finance.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Summary
+
+On May 1, 2023, Level Finance, a decentralized finance (DeFi) protocol, was hacked for $1.1 million in LVL tokens. The attacker [exploited a vulnerability in the protocol's Referral Controller Contract](https://www.halborn.com/blog/post/explained-the-level-finance-hack-may-2023). 
+
+
+## Attackers
+
+The identity of the attacker is unknown.
+
+brc-20:
```
Comment (JediFaust) [inline content/attacks/posts/2023-05-01-Level-Finance.md:16]: Delete the line, or rewrite. ERC20 and BEP20 is a token standards, BSC is a chain.

File: content/attacks/posts/2023-05-01-Level-Finance.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- [0x70319d1c09e1373fc7b10403c852909e5b20a9d5](https://bscscan.com/address/0x70319d1c09e1373fc7b10403c852909e5b20a9d5)
+
+## Losses
+
+The attacker stole 214,000 LVL tokens and swapped LVL to 3,345 BNB, which were worth approximately $1.1 million at the time of the hack.
+
+## Timeline
+
+- **April 24, 2023, 01:52:59 PM +UTC:** [First failed hack attempt](https://bscscan.com/tx/0x95c5b17707294680c06641f253bf79e831ab47b41f41415cc8b93a9ac590363f)
+- **May 1, 2023, 05:50:41 PM +UTC:** [First succesfull hack attempt](https://bscscan.com/tx/0xe1f257041872c075cbe6a1212827bc346df3def6d01a07914e4006ec43027165)
```
Comment (JediFaust) [inline content/attacks/posts/2023-05-01-Level-Finance.md:26]: Please, make sure to perform grammar checks.

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-05-01-Level-Finance.md (hunk 1 above)
Comment (kishoian) [reply content/attacks/posts/2023-05-01-Level-Finance.md:16]: of course, it was an accidental typo

File: content/attacks/posts/2023-05-01-Level-Finance.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ On May 1, 2023, Level Finance, a decentralized finance (DeFi) protocol, was hack
 
 The identity of the attacker is unknown.
 
-brc-20:
+bsc-20:
```
Comment (JediFaust) [inline content/attacks/posts/2023-05-01-Level-Finance.md:16]: Leave as 'BSC' please. [FYI](https://www.google.com/search?q=what+is+bsc+and+bep20+difference)

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, update regarding with recent changes of headers format which can be found in [readme](https://github.com/1712n/dn-institute/tree/main#readme) and update with main branch

File: content/attacks/posts/2023-05-01-Level-Finance.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+date: 2023-05-01
+type of attack: "Protocol exploit"
+target entity: "Level Finance"
+type of entity: "Blockchain"
```
Comment (JediFaust) [inline content/attacks/posts/2023-05-01-Level-Finance.md:5]: Rewrite as 
>target-entities: Silk Road 
entity-types: 
    - Darknet
    - Market
attack-types: 
    - Web Infrastructure Attack
    - Race Condition Exploit

Comment (JediFaust) [reply content/attacks/posts/2023-05-01-Level-Finance.md:5]: Type of Entities for Level Finance are: DeFi, Exchange

File: content/attacks/posts/2023-05-01-Level-Finance.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+date: 2023-05-01
+type of attack: "Protocol exploit"
```
Comment (JediFaust) [inline content/attacks/posts/2023-05-01-Level-Finance.md:3]: Type of attack is 'Smart Contract Exploit'

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-05-01-Level-Finance.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+## Timeline
+
+- **April 24, 2023, 01:52:59 PM +UTC:** [First failed hack attempt](https://bscscan.com/tx/0x95c5b17707294680c06641f253bf79e831ab47b41f41415cc8b93a9ac590363f)
+- **May 1, 2023, 05:50:41 PM +UTC:** [First successful hack attempt](https://bscscan.com/tx/0xe1f257041872c075cbe6a1212827bc346df3def6d01a07914e4006ec43027165)
+- **May 1, 2023, 08:54 PM +UTC:** [The Level Finance team announced the hack on Twitter](https://twitter.com/Level__Finance/status/1653140756540825638)
+- **May 2, 2023:** The DAO has released a [proposal](https://app.level.finance/dao/proposals/0xb057d0796ec3cd09daf01453076a1ced3b9e49173c50abc5d9bf0bd9d8e0e164) asking for votes on how the community should handle the 214K LVL tokens added to circulation by the attack.
+
+## Security Failure Causes
+
+- **Protocol exploit:** The Level Finance hack was made possible by failed precondition checks. In theory, the protocol is designed to allow a user to claim a referral reward once per epoch. However, the protocol lacked checks to ensure that an epoch is not being reused by a claim.
```
Comment (JediFaust) [inline content/attacks/posts/2023-05-01-Level-Finance.md:38]: Security Failure Cause cannot be `Protocol Exploit` change as `Failed Precondition Checks`

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — APPROVED** [review]

**Pull request merged**